alexandria/core/utils.py

```diff
@@ -11,6 +11,9 @@
 
 
 def create_thumbnail(file_: File):
+    if not settings.ALEXANDRIA_ENABLE_THUMBNAIL_GENERATION:
+        return
+
```

Comment:
Now that you bring up the function `create_thumbnail`:
 - it kinda looks like this function it's pretty much a method of `File`
 - we could get rid of `utils.py` moving the function to `File` because nothing left to see

Reply:
Another fair point, done.